zerver/lib/actions.py

```diff
@@ -4921,24 +4922,33 @@ def notify_user_update_custom_profile_data(user_profile: UserProfile,
                                            field: Dict[str, Union[int, str, List[int], None]]) -> None:
     if field['type'] == CustomProfileField.USER:
         field_value = ujson.dumps(field['value'])  # type: Union[int, str, List[int], None]
+        field_rendered_value = None
     else:
         field_value = field['value']
+        field_rendered_value = field['rendered_value']
```

Comment:
Removing this from `if`.. `else` loop would make more sense here. Yes, if field is `User` rendered value will be `None` but it will be `None` in case of `Choice`, `Date` etc. So removing this from `if`..`else` loop would be less confusing.

Reply:
Agreed. Thanks for pointing it out!